src/flatpickr.js

```diff
@@ -1779,6 +1779,14 @@ function Flatpickr(element, config) {
 		return e;
 	}
 
+	function cleanObject(obj) {
```

Comment:
That's one way to solve this, but its far more trivial to simply check the hook value and do something like:

```js
self.config[hooks[i] || []
```